src/ui/table/CombatWidget.cpp

```diff
@@ -519,6 +519,10 @@ CombatWidget::switchCharacterPosition(bool goDown)
 
     setRowAndPlayer();
     pushOnUndoStack();
+
+    m_tableWidget->clearSelection();
+    // selectRow does not work here... not sure why
```

Comment:
It does enable continued shifting, though. Maybe add that to the comment as well.